dev/doc/build-system.dev.txt

```diff
@@ -69,16 +69,16 @@ This file documents internals of the implementation of the make-based
 build system. For what a Coq developer needs to know about the build
 system, see build-system.txt and build-system.dune.md .
 
-.ml4 files
+.mlg files
 ----------
 
-.ml4 are converted to .ml by camlp5. By default, they are produced
+.mlg are converted to .ml by camlp5. By default, they are produced
 in the binary ast format understood by ocamlc/ocamlopt/ocamldep.
```

Comment:
@ppedrot, this section is wrong now, right?  The .mlg are turned into .ml by coqpp, not camlp5, they are not in binary format, and the locations are only approximately right?

Reply:
Yes to all.